nextflow.config

```diff
@@ -48,326 +55,584 @@ if ( params.medaka || params.nanopolish || params.viridian ){
+                    container = '/data/images/aln2type_v0.1.14.sif'
+                }
+                withName:aln2type {
+                    time = '5m'
+                    singularity.enabled = true
+                    container = '/data/images/aln2type_v0.1.14.sif'
+                }
+                withName:viridian {
+                    time = '45m'
+                    singularity.enabled = true
+                    container = '/data/images/viridian_workflow_v0.3.0.img'
+                }
+                withName:viridianPrimers {
+                    time = '45m'
+                    singularity.enabled = true
+                    container = '/data/images/viridian_workflow_v0.3.0.img'
+                }
+                withName:viridianAuto {
+                    time = '45m'
+                    singularity.enabled = true
+                    container = '/data/images/viridian_workflow_v0.3.0.img'
+                }
+                withName:FN4_upload {
+                    time = '1h'
+                    singularity.enabled = true
+                    container = '/data/images/fn4_upload_v0.1.14.sif'
+                }
+                withName:uploadToBucket {
+                    time = '1h'
+                    singularity.enabled = true
+                    container = '/data/images/fn4_upload_v0.1.14.sif'
+                }
+                withName:makeReport {
+                    time = '1h'
+                    singularity.enabled = true
+                    container = "/data/images/artic-ncov2019-illumina_v1.10.0.sif"
+                }
+                withName:getGFF3 {
+                    time = '1h'
+                    singularity.enabled = true
+                    container = "/data/images/bcftools_v0.1.14.sif"
+                }
+                withName:bcftools_csq {
+                    time = '1h'
+                    singularity.enabled = true
+                    container = "/data/images/bcftools_v0.1.14.sif"
+                }
+            }
+        }
+        if (params.cache){
+            singularity.cacheDir = params.cache
+        }
+    }
+    oke {
         process {
+            executor = 'k8s'
             scratch = true
-	    errorStrategy = 'ignore'
+            errorStrategy = { sleep(Math.pow(2, task.attempt) * 200 as long); return 'retry' }
+            maxRetries = 2
+            if(params.myMaxForks){
+                maxForks = params.myMaxForks
+            }
+        }
+        k8s {
+            namespace = 'default'
+            storageClaimName = 'default-nextflow-fss-storage-work-pvc'
+            storageMountPath = '/work'
 
+            serviceAccount = 'default'
+            projectDir = '/work/project'
+            pod = [
+                [label: 'pod-type', value: 'nextflow-worker'],
+                [env: 'NXF_DEBUG', value: '3'],
+                [volumeClaim: 'default-nextflow-fss-storage-data-pvc', mountPath: '/data'],
```

Comment:
Please advise: 

Are there any useful folders under /data that can be shared by multiple nextflow pipelines, if each of them has unique execution ID and batch name?